board.py

```diff
@@ -1,33 +1,90 @@
 class Board:
-    board_state = []
+    def __init__(self):
+        self.board_state = []
 
     def new_game(self, numrows=3):
         board_area = numrows*numrows
-        new_board = [" "] * board_area
+        new_board = [""] * board_area
         self.board_state = new_board
 
-def spot_exists(current_state, spotChoice):
-    if (spotChoice - 1) >= 0 and spotChoice <= len(current_state):
-        return True
-    else:
-        return False
+class WinningCombos:
```

Comment:
What do you gain by adding this class and the ones below it?

Reply:
I gain the ability to find all possible winning combinations for any board size.

Reply:
I meant more like why does this class even exist. Does it make sense to separate this out from the rest of the board? Does it deserve to be it's own class?